src/testing/code-debugging.md

```diff
@@ -677,12 +693,18 @@ used to reduce how much needs to be repainted.
 
 ### Focus tree
 
-To debug a focus or shortcut issue, you can dump the focus tree
-using [`debugDumpFocusTree()`][].
+To debug a focus or shortcut issue, dump the focus tree
+using the [`debugDumpFocusTree()`][] method.
```

Comment:
Since it's not a class or instance function, it is not a method, just a function.

```suggestion
using the [`debugDumpFocusTree()`][] function.
```